Orange/widgets/visualize/owscatterplot.py

```diff
@@ -372,6 +372,7 @@ def set_subset_data(self, subset_data):
                 self.warning("Data subset does not support large Sql tables")
                 subset_data = None
         self.subset_data = self.move_primitive_metas_to_X(subset_data)
+        self.controls.graph.alpha_value.setEnabled(subset_data is None)
```

Comment:
Could you add some tests, specifically for multilevel access, checking both the cases when the "graph" is declared as settings provider and when it is not?

(tests should **not** be a part of scatterplot tests, I am just writing a comment here because it is convenient :))